products/redhat-openjdk.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Red Hat OpenJDK
+category: lang
+tags: java-distribution redhat
+iconSlug: openjdk
+permalink: /redhat-openjdk
+versionCommand: java -version
+releasePolicyLink: https://access.redhat.com/articles/1299013
+releaseDateColumn: true
+eolColumn: Support
+releaseLabel: "OpenJDK __RELEASE_CYCLE__"
+
+# Based on searching for "openjdk runtime environment" and
+# "openjdk devel" at
+# https://access.redhat.com/downloads/content/package-browser
+# This is not great, since it is quite impossible to match
+# identifiers to a release cycle.
+identifiers:
+    - purl: pkg:redhat/java-1.6.0-openjdk
+    - purl: pkg:redhat/java-1.7.0-openjdk
+    - purl: pkg:redhat/java-1.7.0-openjdk-headless
+    - purl: pkg:redhat/java-1.8.0-openjdk
+    - purl: pkg:redhat/java-1.8.0-openjdk-debug
+    - purl: pkg:redhat/java-1.8.0-openjdk-headless
+    - purl: pkg:redhat/java-1.8.0-openjdk-headless-debug
+    - purl: pkg:redhat/java-11-openjdk
+    - purl: pkg:redhat/java-11-openjdk-debug
+    - purl: pkg:redhat/java-11-openjdk-headless-debug
+    - purl: pkg:redhat/java-1.6.0-openjdk-devel
+    - purl: pkg:redhat/java-1.7.0-openjdk-devel
+    - purl: pkg:redhat/java-1.8.0-openjdk-devel
+    - purl: pkg:redhat/java-1.8.0-openjdk-devel-debug
+    - purl: pkg:redhat/java-11-openjdk-devel
+    - purl: pkg:redhat/java-11-openjdk-devel-debug
+
+releases:
```

Comment:
Is there a way to automate patch release update ?

Reply:
I looked at a few sources, and it doesn't look like it. Neither pkgs.org nor repology seems to index RHEL, so the versions are incorrect (they use Centos, stops getting updates much earlier).

For eg, https://pkgs.org/download/java-1.6.0-openjdk or https://repology.org/project/openjdk/versions